File: 3월-8월/8월/20210815/정충일/2.로또의 최고 순위와 최저 순위.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> lottos, vector<int> win_nums) {
    vector<int> answer;
    int zero_count = 0;
    int win_count = 0;
    for(int i=0;i<lottos.size();i++)
        if(lottos[i] == 0) zero_count++; // 0이 몇개인가!
    //같은 숫자 체킹
    for(int i=0;i<win_nums.size();i++) 
        for(int j=0;j<win_nums.size();j++) 
            if(lottos[i] == win_nums[j]) win_count++;
    //범위 안에 들기 위해 count up
    if(win_count == 0) {
        win_count++;
        if(zero_count==6) zero_count--; //예외
    }
    //결과 입력
    answer.push_back(7-(win_count+zero_count));
    answer.push_back(7-win_count);
    return answer;
}

File: 3월-10월/8월/20210815/정충일/2.로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    answer = []
    zero_count = 0
    win_count = 0
    for i in range(len(lottos)):
        if(not lottos[i]): zero_count+=1 #0이 몇개인가!
    for i in range(len(win_nums)):
        for j in range(len(win_nums)):
            if(lottos[i] == win_nums[j]): win_count+=1 #같은 숫자 체킹
    if(not win_count): #범위 안에 들기 위해 count up
        win_count+=1
        if(zero_count == 6): zero_count-=1 #에외
    answer.append(7-(win_count+zero_count)) #결과 입력
    answer.append(7-win_count)
    return answer